Makefile

```diff
@@ -0,0 +1,25 @@
+##################
+# Pre-requisites
+# 	- Foundry
+# 	- pnpm
+#	- cargo-clippy
+##################
+
+# Check for tests to pass
+.PHONY: test
+test:
+	cargo test
+
+# Run before sending PRs
+.PHONY: reportgen
+reportgen:
+	cargo fmt
+	cargo clippy -- -D warnings 
+	cli/reportgen.sh
+
```

Comment:
This is not that stable yet - because we might have to say `cargo +nightly fmt`  Context https://github.com/Cyfrin/aderyn/pull/424